### PR TITLE
Strings methods: `startswith()`

### PR DESCRIPTION
**Pull request by Abdelrahman-Kh-Fouad**

### CODE REVIEW

**Comment by Abdelrahman-Kh-Fouad** [comment]
@certik @czgdp1807 
Ready for review

File: integration_tests/test_str_attributes.py [hunk 1]
```diff
@@ -25,6 +25,21 @@ def strip():
     assert "     AASAsaSas    " .lstrip() == "AASAsaSas    "
     assert "     AASAsaSas    " .strip() == "AASAsaSas"
 
+def startswith():
+    s: str
+    s = "   empty"
+    assert s.startswith("  ") == True
+    assert "  @" .startswith(" ") == True
+    assert "   emptyAaaaAABBbbbbBB" .startswith(s) == True
```
Comment (czgdp1807) [inline integration_tests/test_str_attributes.py:33]: Let's try giving something in the middle portion of the string such as `AABB` in `startswith` and see if it returns `False`. Also give something like `emptyAaaX` as input to `startswith`.

**Review by czgdp1807 — CHANGES_REQUESTED** [review]

File: integration_tests/test_str_attributes.py (hunk 1 above)
Comment (Abdelrahman-Kh-Fouad) [reply integration_tests/test_str_attributes.py:33]: i added them and edit those corner cases in functions, i think it's ready now.

**Review by czgdp1807 — APPROVED** [review]
LGTM. @certik Should we merge this?

**Comment by czgdp1807** [comment]
Ah! Resolve the conflicts @Abdelrahman-Kh-Fouad.

**Comment by Abdelrahman-Kh-Fouad** [comment]
@czgdp1807 Done!

**Pull request merged**